fyle_accounting_mappings/views.py

```diff
@@ -242,11 +242,20 @@ def get(self, request, *args, **kwargs):
                 filters['source__active'] = True
 
             mapped_attributes_count = Mapping.objects.filter(**filters).count()
+            activity_mapping = None
 
             if source_type == 'CATEGORY':
                 activity_attribute_count = ExpenseAttribute.objects.filter(
                     attribute_type='CATEGORY', value='Activity', workspace_id=self.kwargs['workspace_id'], active=True).count()
-                if activity_attribute_count:
+                
+                if app_name in ('NETSUITE', 'INTACCT', 'SAGE_300_CRE', 'DYNAMICS_365_BUSINESS_CENTRAL'):
```

Comment:
this seems incorrect, check above conditions

Reply:
app names

Reply:
these are right values
  BAMBOO_HR = 'BambooHR',
  QBD = 'QuickBooks Desktop',
  TRAVELPERK = 'TravelPerk',
  INTACCT = 'Sage Intacct',
  SAGE300 = 'Sage 300 CRE',
  QBO = 'QuickBooks Online',
  BUSINESS_CENTRAL = 'Dynamics 365 Business Central',
  NETSUITE = 'NetSuite',
  XERO = 'Xero'

pls test it from app as well